### PR TITLE
[MIG] shopinvader_assortment: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Review by Cedric-Pigeon — APPROVED** [review]
Code review

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1 @@
-# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+server-tools
```
Comment (simahawk) [inline oca_dependencies.txt:1]: this is not needed

File: shopinvader_assortment/models/shopinvader_backend.py [hunk 2]
```diff
@@ -26,9 +26,7 @@ def _autobind_product_from_assortment(self):
         unbinding_wizard_obj = self.env["shopinvader.variant.unbinding.wizard"]
         assortment_domain = self.product_assortment_id._get_eval_domain()
         assortment_products = product_obj.search(assortment_domain)
-        variants_binded = shopinvader_variant_obj.search(
-            [("backend_id", "=", self.id)]
-        )
+        variants_binded = shopinvader_variant_obj.search([("backend_id", "=", self.id)])
```
Comment (simahawk) [inline shopinvader_assortment/models/shopinvader_backend.py:29]: s/binded/bound

File: test-requirements.txt [hunk 3]
```diff
@@ -8,7 +8,6 @@ vcrpy-unittest
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
 
-
```
Comment (simahawk) [inline test-requirements.txt:11]: this file must be cleaned up

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
please remove the last merge commit as well :pray:

**Comment by simahawk** [comment]
Replaced by #973

Sorry guys, is getting insane w/ pending merges :sweat_smile: and I kind of need to move faster :wink: 

I've done what commented on #863 to clean up.

**Pull request closed without merging**